ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -146,6 +146,7 @@ descriptor:
   id: "An identification or identifier."
   inlet: "Area of media entrance."
   inhibit: "Prohibit from executing a function or starting an operation."
+  injection: "A process that introduces a substance, such as fuel, water, or air, into a system or engine by using pressure or vacuum."
```

Comment:
please tailor this to be more HVAC focused, something like  "A process that introduces a substance, such as refrigerant, water, or air, into a system by using pressure or vacuum."